torch/_inductor/lowering.py

```diff
@@ -2467,6 +2467,8 @@ def load_from_output(x, y):
         def index_range_condition(index_range):
             i, lb, ub = index_range
             i = ops.index_expr(i, torch.int32)
+            lb = ops.index_expr(lb, torch.int64)
+            ub = ops.index_expr(ub, torch.int64)
```

Comment:
After switching to `specialize_int=False`, these can be SymPy exprs. So I wrap them into `index_expr`.

Reply:
sure would be nice if we had a type checker for this sort of thing 🤔